files/bind/named.conf

```diff
@@ -6,10 +6,31 @@ include "/etc/bind/named.conf.options";
 include "/etc/bind/named.conf.local";
 
 logging {
-    category default { default_stderr; };
-    category general { default_stderr; };
-    category config { default_stderr; };
-    category network { default_stderr; };
-    category queries { default_stderr; };
-    category security { default_stderr; };
+    channel general {
```

Comment:
leave the `std_err` here for this file (in other words do not revert this guy).  This will have us logging to std err which is good for `docker logs`